EnglishFilter/sections/antiadblock.txt

```diff
@@ -634,6 +634,8 @@ faucetcoins.org,dogecrypto.club,getdogecoins.com#%#//scriptlet("set-constant", "
 masihbelajar.com#%#//scriptlet("set-constant", "fabActive", "false")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59106
 @@||vanis.io/ads.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/61800
+||zkillboard.com/js/common.js?t=202008181502 
```

Comment:
```suggestion
||zkillboard.com/js/common.js
```

Reply:
found out it also breaks the search feature, obviously 
reworking